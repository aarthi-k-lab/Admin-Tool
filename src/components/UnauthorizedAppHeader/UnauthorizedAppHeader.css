.header {
  align-items: center;
  background-color: rgb(0, 187, 215);
  box-sizing: border-box;
  display: flex;
  flex-wrap: nowrap;
  height: 54px;
  padding: 0.5em 1em;
  width: 100%;
}

.spacer {
  flex-grow: 1;
}
