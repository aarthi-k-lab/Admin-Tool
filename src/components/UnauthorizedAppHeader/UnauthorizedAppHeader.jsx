import React from 'react';

import './UnauthorizedAppHeader.css';

function Header() {
  return (
    <header styleName="header">
      <img alt="logo" src="/static/img/logo.png" />
      <span styleName="spacer" />
    </header>
  );
}

export default Header;
