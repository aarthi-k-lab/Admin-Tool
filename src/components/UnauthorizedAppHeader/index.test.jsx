import React from 'react';
import renderer from 'react-test-renderer';
import UnauthorizedAppHeader from '.';

describe('<UnauthorizedAppHeader />', () => {
  test('snapshot test', () => {
    const tree = renderer
      .create(<UnauthorizedAppHeader />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
