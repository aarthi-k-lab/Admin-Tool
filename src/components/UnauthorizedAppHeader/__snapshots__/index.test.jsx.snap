// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<UnauthorizedAppHeader /> snapshot test 1`] = `
<header
  className="components-UnauthorizedAppHeader-UnauthorizedAppHeader__header--1Qm9B"
>
  <img
    alt="logo"
    src="/static/img/logo.png"
  />
  <span
    className="components-UnauthorizedAppHeader-UnauthorizedAppHeader__spacer--1Ryki"
  />
</header>
`;
